src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -50,12 +78,23 @@ public CommandResult(String feedbackToUser, PanelName panelName) {
 
     /**
      * Constructs a {@code CommandResult} with the specified {@code feedbackToUser},
-     * and other fields set to their default value.
+     * and other fields set to their default value. Meant for most MooLah commands
      */
     public CommandResult(String feedbackToUser) {
         this(feedbackToUser, false, false, false, false, PanelName.CURRENT);
     }
 
+    /**
+     * Constructs a {@code CommandResult} with the specified {@code feedbackToUser},
+     * and other fields set to their default value. Meant for Help and Bye commands.
+     */
+    public CommandResult(String feedbackToUser, boolean showHelp, boolean exit, boolean statistic) {
+        this(feedbackToUser, showHelp, exit, statistic, false, null, null, null, null);
```

Comment:
new issues:
panelName field  should be PanelName.CURRENT , or another PanelName. Should not be null.

Reply:
not urgent to resolve atm